rotkehlchen/db/eth2.py

```diff
@@ -20,6 +21,10 @@
 log = RotkehlchenLogsAdapter(logger)
 
 WITHDRAWALS_RECHECK_PERIOD = 3 * 3600 * 1000  # 3 hours in milliseconds
+ETH_STAKING_EVENTS = (
```

Comment:
Isn't this duplicating almost all of `ETH_STAKING_EVENT_JOIN` which is defined in `db/filtering.py`?
Please do not do that.